### PR TITLE
Add information on how the gRPC project hands out commit access.

### PR DESCRIPTION
**Pull request by hsaliak**
Update CONTRIBUTING.md with a note on obtaining Commit Access to gRPC

### CODE REVIEW

**Comment by srini100** [comment]
Looks fine since we are trying to keep this short and simple. Should we at least mention a way to reach out if someone wants committer access? [governance.md](https://github.com/grpc/grpc-community/blob/master/governance.md) asks to create an issue in the repo if one wants to become a reviewer. May be we should remove that section.

I'll let @jayantkolhe approve this.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -110,5 +110,16 @@ How to get your contributions merged smoothly and quickly.
 - Exceptions to the rules can be made if there's a compelling reason for doing
   so.
 
-
+## Obtaining Commit Access
+We grant Commit Access from time to time to contributors based on the following criteria:
+* Sustained contribution to the gRPC project.
```
Comment (jayantkolhe) [inline CONTRIBUTING.md]: remove "from time to time"

Comment (hsaliak) [reply CONTRIBUTING.md]: ack, ptal.

**Comment by srini100** [comment]
Spoke to @jayantkolhe offline. Looks good. We can revise it later if needed.

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* One or more commits are not associated with a GitHub user. The email used in the Git commit must match a verified email address in a GitHub user profile. For more information, please see https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/

_Regards,
CLA GitHub bot_

**Pull request merged**